bench/criterion_benchmark.rs

```diff
@@ -29,6 +29,7 @@ const BENCH_NAMES: &[&str] = &[
     "operations_with_data_structures_benchmarks",
     "uint256_integration_benchmark",
     "set_integration_benchmark",
+    "poseidon_integration_benchmark",
```

Comment:
You should add it to `iai_benchmark.rs`.

Reply:
Nice catch! Added!